### DISCUSSION TITLE
signal power (-M D) shown in db on localhost:8100

### DISCUSSION THREAD

**Discussion by OE1MWW** [jvde-github/AIS-catcher#155] [General]
Hi all,

I compared, using a SDR-RTL dongle and a MSI.SDR to check a signal sent by a 'Base Station'

in the RSSI field of localhost:8100 I see

- SDR-RTL -16db
- MSI.SDR -64 db 

that would mean, this is a signal difference of ~ 50db, or, with other words, the received signal 
voltage is 316 times higher on the RTL-SDR compared to the MSI.SDR (?),  

Btw. 'dbm' is a relation to "power of 1 milliwat on 50 ohms", but 'db' is just a relational number 
- what is the reference ? 

anyone has a comment on this please ?

**Comment by jvde-github** [comment]
Hi, just a note on the number shown which might have a different interpretation from what you expect. 
It is calculated as 10 * log10(signal level), where signal level is the average over the AIS signal of the sample value squares. So the reference is 1 for now. 

A RTL_SDR produces numbers between -128 and +127 which is scaled to -1 to 1. For an SDRPlay we read in integers which we scale by dividing by 32768 which already gives a factor difference as the SDRlay samples at 14 bits (I guess a factor 8 ~2 ** 3, 2 bits difference plus one for square in the formula).

So, outright difficult to compare devices. Sorry for confusion.

**Reply by jvde-github** [reply]
I just googled the MSI SDR, thought it was a SDRPlay but it is a different dongle with 12 bits sampling. What driver are you using for this?

if you would have a device that actually samples at 8 bits, you would have a max of 10*math.log10((128/32768)**2) = -48 dB.....

**Reply by OE1MWW** [reply]
The SDRPlay RSP1 and all the MSI.SDR using the same MSi2500/MSi001 'MIRICS' chips with 12 bit sampling.
On the SDRPlay RSP1A they claim 14bit sampling, but make no reference or info, how they achived that.
RSP1A - ADC Sample Rates 2MSPS – 10.66MSPS, 
ADC Number of Bits: 
14bit 2MSPS – 6.048MSPS
12bit 6.048MSPS – 8.064MSPS
10bit 8.064MSPS – 9.216MSPS
8bit >9.216MSPS

I was just puzzled by the indicated difference of 50db 'RTL-SDR vs MSI.SDR' in the RSSI indication.
AFIK the 'RSSI' is a positive number and indicates a signal strength number (device depending, dimensionless), mostly used in mobile phones. And to make it more complicated, on some devices the info is shown in 'ASU' (Arbitrary Strength Unit), also dimensionless or in 'dBm', that one is based on 1mW/50ohms.

Anyway, your software is well done, makes fun and numbers are just numbers

**Comment by OE1MWW** [comment]
compared the nooelec V5 to the MSI.SDR (SDRPlay RSP1 clone)

on the nooelec V5 I see the RSSI of two base stations: A -15db, B -30db
on the MSI.SDR I see just one, the A with -53db, B is not shown in the list & map

Listeneing to AM or FM stations on VHF, I could not see (hear) any difference
of the two devices. 

So, can someone please point me to settings in AISCatcher of the SDRPlay device
